src/contrib/cluster/Akka.Cluster.Sharding/ClusterSharding.cs

```diff
@@ -111,10 +111,10 @@ public Implementation(int maxNumberOfShards, Func<Msg, string> entityIdExtractor
                 _messageExtractor = messageExtractor;
             }
 
-            public override string EntityId(Msg message)
+            public override string? EntityId(Msg message)
```

Comment:
We applied `nullability` to the `IMessageExtractor` interface for these methods some time ago, but never applied it to the `HashCodeMessageExtractor` class that implements it - first that here and you'll see it show up in the API approvals as well.